jina/peapods/runtimes/asyncio/rest/app.py

```diff
@@ -199,13 +200,13 @@ async def delete_api(body: JinaDeleteRequestModel):
 
     async def result_in_stream(req_iter):
         """
-        Collect the protobuf message converts protobuf message to a dictionary.
+        Collect the protobuf message converts protobuf messages to strings.
 
         :param req_iter: request iterator
         :yield: result
         """
         async for k in servicer.Call(request_iterator=req_iter, context=None):
-            yield MessageToDict(k)
+            yield json.dumps(MessageToDict(k))
```

Comment:
why do we want a string and not a dict?

Reply:
I think the proper way is to return a `dictionary` and let the processing into documents to the client.

Reply:
This is also a potential breaking change if someone was using its own client?

Reply:
Actually the api endpoint is broken on master. This one of the fix I did
I could not get it to run returning dictionaries in the StreamingResponse.
Reproducting steps:
```
from jina import Document
with Flow(restful=True, port_expose=8089).add() as f:
    f.block()
```
Go to `http://localhost:8089/docs#/CRUD/index_api_index_post` -> try out -> execute

Reply:
```suggestion
            yield MessageToJson(k)
```

Reply:
Also, please add `media_type` to all `StreamingResponse`
```python
return StreamingResponse(result_in_stream(request_generator(**bd)),
                                 media_type="application/json")
```